block/meta-iosched.c

```diff
@@ -250,3 +253,12 @@ void update_stats(struct request_queue *q) {
 		clean_up(group);
 	}
 }
+
+void init_my_timer(void) {
+	struct timer_list my_timer;
```

Comment:
AFAICS, it shouldn't be allocated on stack, because it would be freed as soon as function returns.